examples/tf/te_ppo_metaworld_ml1_push.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+"""This is an example to train Task Embedding PPO with PointEnv."""
+# pylint: disable=no-value-for-parameter
+import click
+from metaworld.benchmarks import ML1
+import tensorflow as tf
+
+from garage import wrap_experiment
+from garage.envs import GarageEnv, normalize
+from garage.envs.multi_env_wrapper import MultiEnvWrapper
+from garage.experiment import LocalTFRunner
+from garage.experiment.deterministic import set_seed
+from garage.np.baselines import LinearMultiFeatureBaseline
+from garage.sampler import LocalSampler
+from garage.tf.algos import TEPPO
+from garage.tf.algos.te import TaskEmbeddingWorker
+from garage.tf.embeddings import GaussianMLPEncoder
+from garage.tf.policies import GaussianMLPTaskEmbeddingPolicy
+
+
+@click.command()
+@click.option('--seed', default=1)
+@click.option('--n_epochs', default=600)
+@click.option('--batch_size_per_task', default=1024)
+@wrap_experiment
+def te_ppo_ml1_push(ctxt, seed, n_epochs, batch_size_per_task):
+    """Train Task Embedding PPO with PointEnv.
+
+    Args:
+        ctxt (garage.experiment.ExperimentContext): The experiment
+            configuration used by LocalRunner to create the snapshotter.
+        seed (int): Used to seed the random number generator to produce
+            determinism.
+        n_epochs (int): Total number of epochs for training.
+        batch_size_per_task (int): Batch size of samples for each task.
+
+    """
+    set_seed(seed)
+    envs = [GarageEnv(normalize(ML1.get_train_tasks('push-v1')))]
+    env = MultiEnvWrapper(envs, mode='del-onehot')
+
+    latent_length = 2
+    inference_window = 6
+    batch_size = batch_size_per_task
```

Comment:
this seems inconsistent because you have 50 tasks but are only using 1 `batch_size_per_task` worth of samples?

Reply:
I think the 50 internal goals are obscure to the algorithm. In case of ML1, the algorithm should just see one task, so it uses only 1 `batch_size_per_task`.